### ISSUE TITLE
[wg/i18n-core, ig/i18n] i18n WG & IG rechartering 2024

### ISSUE THREAD

**Issue by xfq** [w3c/strategy#468]
New charter proposal, reviewers please take note.

# Charter Review

[WG Charter](https://w3c.github.io/i18n-activity/i18n-wg/charter.html)

[IG Charter](https://w3c.github.io/i18n-activity/i18n-ig/charter.html)

If applicable:

[diff from previous WG charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FInternational%2Fgroups%2Fwg%2Fcharter-2021.html&doc2=https%3A%2F%2Fw3c.github.io%2Fi18n-activity%2Fi18n-wg%2Fcharter.html)

[diff from previous IG charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FInternational%2Fgroups%2Fig%2Fcharter-2021.html&doc2=https%3A%2F%2Fw3c.github.io%2Fi18n-activity%2Fi18n-ig%2Fcharter.html)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* Existing 
 - [x] Existing WG recharter 
 - [x] Existing IG recharter

Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, security, and TAG. **Also add a "card" for this issue to the [Strategy Funnel](https://github.com/w3c/strategy/projects/2).**

Communities suggested for outreach:
none

Known or potential areas of concern:
none

Where would charter proponents like to see issues raised?
this strategy funnel issue

Anything else we should think about as we review? 
no

/cc @aphillips @duerst

**Comment by simoneonofri** [comment]
no comments on the Security side :)

as we said let's keep aligned on how to do the various reviews

**Comment by himorin** [comment]
no comment or request from i18n

**Comment by ruoxiran** [comment]
no comment or request from APA for those two charters.

**Comment by plehegar** [comment]
In the WG charter, the changelog can be updated to remove a past change about the chair.

The IG and WG charters are not aligned with the charter template. For example:
- the link and last two words for the Patent Disclosures deviate from the charter template. Please align with the charter template for those bits.
- We updated recently the links to the patent policy and process document in the charter template, please align. 
- The Communication sections don't mention the W3C Process nor the Code of Conduct.

Please align those charters with the charter template.

You're asking for 3 years instead of the usual 2 years for the duration of the WG/IG. Is that intentional ? I won't object to it because those 2 Groups are core to our mission but we may get pushback from the AC.

**Comment by xfq** [comment]
Thanks, @plehegar. Please see comments below.

> In the WG charter, the changelog can be updated to remove a past change about the chair.

Done.

> The IG and WG charters are not aligned with the charter template. For example:
> 
> * the link and last two words for the Patent Disclosures deviate from the charter template. Please align with the charter template for those bits.

I have fixed the WG charter.

For the IG charter, it doesn't seem to have a licensing information page. https://www.w3.org/groups/ig/i18n/ipr/ is 404. Is it because the IG is essentially a mailing list?

> * We updated recently the links to the patent policy and process document in the charter template, please align.

Updated.

> * The Communication sections don't mention the W3C Process nor the Code of Conduct.

I compared the [charter](https://w3c.github.io/i18n-activity/i18n-wg/charter.html#communication) and the [template](https://w3c.github.io/charter-drafts/charter-template.html#communication), but didn't find see the W3C Process or the Code of Conduct mentioned. Am I missing something?

> Please align those charters with the charter template.

Done.

> You're asking for 3 years instead of the usual 2 years for the duration of the WG/IG. Is that intentional ? I won't object to it because those 2 Groups are core to our mission but we may get pushback from the AC.

I'm asking for 3 years, because the work of the i18n groups is relatively stable and the scope/deliverables generally does not change much. Also, the last charter period was also 3 years (2021-2024).

**Comment by siusin** [comment]
The Success Criteria section is missing from the IG charter.

**Comment by xfq** [comment]
> The Success Criteria section is missing from the IG charter.

Thanks. Added now: https://w3c.github.io/i18n-activity/i18n-ig/charter.html#success-criteria

**Comment by xfq** [comment]
https://lists.w3.org/Archives/Member/w3c-ac-members/2024OctDec/0020.html
